cookbooks/ros2_windows/recipes/opencv.rb

```diff
@@ -1,5 +1,5 @@
 seven_zip_archive 'open_cv_zip' do
-  path 'C:\\opencv'
+  path 'C:\\'
```

Comment:
this is required because the archive contains an `opencv` directory and so was being installed to `c:\opencv\opencv` correct?

Reply:
Yup, that is correct